docs/developer/products/api.mdx

```diff
@@ -776,3 +776,243 @@ Here's a more complex GraphQL query that combines all of the above information:
   }
 }
 ```
+
+## Create Products
+
+:::info  
+To create products and variants, you need the `MANAGE_PRODUCTS` permission.  
+::: 
+
+Before you can create a product, ensure the following exist:
+
+- Product Type – Defines whether the product is shippable (physical vs. digital) and which attributes (e.g., size, color) it supports.
+- Category – Organizes products into a hierarchical tree structure, often used for navigation in the storefront (e.g., "Clothing > Men > Shirts").
+- Collection (Optional) – Groups products for marketing purposes or curated storefront sections (e.g., “Summer Sale” or “Best Sellers”).
+- Channel – Controls the product's visibility, availability, pricing, and publication status across different sales channels (e.g., online store, marketplace).
+- Warehouse – Required to track stock levels and assign inventory to product variants.
```

Comment:
Suggestion (non-blocking): Just an idea - maybe we can link the corresponding documentation links that describe each of those in more details.

Reply:
I will put this in my backlog, because for now I can only link the channel. In fact, we do not have good docs related to the product type, category, collection and I am currently working on the warehouse section.